# file: /usr/lib/python3/dist-packages/pytz/exceptions.py
# hypothesis_version: 6.140.2

['AmbiguousTimeError', 'InvalidTimeError', 'NonExistentTimeError', 'UnknownTimeZoneError']